PopPUNK/visualise.py

```diff
@@ -9,6 +9,16 @@
 import numpy as np
 import scipy.sparse
 
+# GPU support
+try:
+    import cugraph
+    import cudf
+    import cupy as cp
+    from numba import cuda
```

Comment:
Add all these packages to docs under GPU use instructions

Reply:
Added in 664a478.